test/Utilities/StyleMocking.js

```diff
@@ -0,0 +1,40 @@
+import assert from 'assert'
+
+// Domino doesn't seem to slurp up css class properties if we set them in our fixture html.
```

Comment:
Hmm not sure I follow... the comment describes (perhaps poorly haha) how I'm using this file for testing, but perhaps proper method comments would be better?